src/Umbraco.Core/Packaging/PackagesRepository.cs

```diff
@@ -21,6 +21,7 @@ namespace Umbraco.Cms.Core.Packaging
     /// <summary>
     /// Manages the storage of installed/created package definitions
     /// </summary>
+    [Obsolete("Packages has now been moved to the database instead of local files, please use CreatedPackageSchemaRepository instead")]
```

Comment:
```suggestion
    [Obsolete("Packages have now been moved to the database instead of local files, please use CreatedPackageSchemaRepository instead")]
```